packages/loot-core/bin/build-browser.js

```diff
@@ -0,0 +1,78 @@
+#!usr/bin/env node
+const { fsUtil, shellUtil, webpackUtil, Path } = require('../../../bin/utils');
+
+const migrationUtil = require('./migration-util');
+
+const ROOT = Path.join(process.cwd(), '/bin'); //Make path consistent with bash
```

Comment:
I think it would make more sense to strip off the `bin` here and remove a `../` from the things that reference `ROOT`.